lib/katello/engine.rb

```diff
@@ -16,6 +16,9 @@ class Engine < ::Rails::Engine
 
       require 'katello/middleware/event_daemon'
       app.middleware.use(Katello::Middleware::EventDaemon)
+
+      require 'katello/middleware/organization_created_enforcer'
+      app.middleware.use(Katello::Middleware::OrganizationCreatedEnforcer)
```

Comment:
Does the middleware approach work better than controller before-action thingies?

Reply:
I like it better because the middleware approach will only check to run the task **about once per process** (in theory) unless there is a huge flood of requests coming in at startup time. Will be good to check what happens when candlepin is down and it needs to run the task though.

On the flip side, the Concern approach will check the org for each request at least how i've written it.